android/app/src/main/java/com/mattermost/helpers/database_extension/General.kt

```diff
@@ -56,7 +56,7 @@ fun DatabaseHelper.getDatabaseForServer(context: Context?, serverUrl: String): W
         defaultDatabase!!.rawQuery(query, arrayOf(serverUrl)).use { cursor ->
             if (cursor.count == 1) {
                 cursor.moveToFirst()
-                val databasePath = cursor.getString(0)
+                val databasePath = String.format("file://%s", cursor.getString(0))
```

Comment:
This is unrelated to this fix, but needed.

While testing, I saw that `getDatabaseForServer` was throwing exceptions because it was not able to find the file. This only happened to server databases, and not the app database. After looking at the difference between the the two calls, I saw that the app database route always had the `file://` prefix while the server databases didn't have such prefixes.

After this change, everything started to work again.